core/carts/common/Gade-Carts-RTC-File_IO.adb

```diff
@@ -0,0 +1,145 @@
+with Ada.Calendar.Conversions;   use Ada.Calendar.Conversions;
+with Ada.Exceptions;             use Ada.Exceptions;
+with Ada.Text_IO;
+with Interfaces.C;
+
+package body Gade.Carts.RTC.File_IO is
+
+   procedure To_Counter_Registers
+     (C_Data    : Counter_Data;
+      Registers : out Counter_Registers)
+   is
+      pragma Unsuppress (Overflow_Check);
+
+      Days_Top : constant Unsigned_32 := Unsigned_32 (C_Data.Days_Top);
+      Days : constant Unsigned_32 :=
+        C_Data.Days_Low + Days_Top * Days_Low_Cardinality;
+   begin
+      Ada.Text_IO.Put_Line ("To_Counter_Registers.SS" & C_Data.Seconds'Img);
```

Comment:
Printing related code should go to its own package, might still be valuable for debugging/logging in the future even if not required.